2-mandatory/1-create-functions.js

```diff
@@ -3,15 +3,20 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
-}
+function first5(array) {
+  const size=5;
+  return array.slice(0,size);
+  
+  }
 
 /*
 Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the same elements, except sorted.
 */
-function sortArray() {
+function sortArray(arr) {
+  let newArr=arr.slice(0);
+  return newArr.sort();
 }
```

Comment:
Correct. (1) Make a copy of the array. (2) sort it. (3) return the sorted array. This also works and is a bit more "modern":
```
return [...arr].sort();
```
The three dots is called the "spread operator". When used inside "[" and "]", a new array is created, which here is then sorted and returned. It's a very shorthand way of doing the 3 steps above!